app/views/govuk_publishing_components/audit/show.html.erb

```diff
@@ -39,98 +39,105 @@
           </div>
         </details>
 
-        <div class="gem-c-accordion" data-module="gem-c-accordion" id="accordion-with-summary-sections">
-          <% @applications.each_with_index do |application, index| %>
-            <div class="gem-c-accordion__section ">
-              <div class="gem-c-accordion__section-header">
-                <h2 class="gem-c-accordion__section-heading">
-                  <span class="gem-c-accordion__section-button" id="accordion-with-summary-sections-heading-<%= index %>">
-                    <%= application[:name] %>
-                  </span>
-                </h2>
-                <div class="gem-c-accordion__section-summary govuk-body" id="accordion-with-summary-sections-summary-<%= index %>">
-                  <% if application[:application_found] %>
-                    Warnings:
-                    <% if application[:warning_count] > 0 %>
-                      <strong class="govuk-tag govuk-tag--red"><%= application[:warning_count] %></strong>
-                    <% else %>
-                      <%= application[:warning_count] %>
-                    <% end %>
-                  <% else %>
-                    <strong class="govuk-tag govuk-tag--red">Application not found</strong>
-                  <% end %>
-                </div>
-              </div>
-              <div id="accordion-with-summary-sections-content-<%= index %>" class="gem-c-accordion__section-content" aria-labelledby="accordion-with-summary-sections-heading-<%= index %>">
-                <% if application[:application_found] %>
-                  <% application[:warnings].each do |warning| %>
-                    <p class="govuk-body">
-                      <strong class="govuk-tag">Warn</strong>
-                      <strong><%= warning[:component] %></strong> - <%= warning[:message] %>
-                    </p>
-                  <% end %>
+        <% application_items = [] %>
+        <% @applications.each_with_index do |application, index| %>
+          <%
+            summary = '<strong class="govuk-tag govuk-tag--red">Application not found</strong>'
```

Comment:
I don't think you use `index` anywhere - if that's the case, this could be rejigged to use `.map` directly to `application_items`

eg:

```suggestion
        <% application_items = @applications.map do |application| %>
```

Reply:
I don't need the `index` but I'd need to change quite a bit more to make this work as inside this loop it adds things onto the `application_items` array, for output by the component. It's probably doable but more of an optimisation than I want to get into right now.

Reply:
Okaydokey - approved since using map or each doesn't make a difference what's being output.